test/centering.jl

```diff
@@ -14,6 +13,13 @@
         @test yc isa CenteredTerm
         @test yc.center == mean(data.y)
         @test modelcols(yc, data) == data.y .- mean(data.y) == data.y .- yc.center
+
+        @testset "alternative center function" begin
+            xc = concrete_term(term(:x), data, Center(median))
+            @test xc isa CenteredTerm
+            @test xc.center == median(data.x)
+            @test modelcols(xc, data) == data.x .- median(data.x) == data.x .- xc.center
```

Comment:
in this case median is going to give you the same as mean right?  maybe use something like `first` to really get wacky?

Reply:
That's a good idea.